File: CSharp/48. Other Algorithms/02. TrianglePoints/Point.cs
namespace _02.TrianglePoints
{
    public struct Point
    {
        public Point(double pointX, double pointY)
            : this()
        {
            this.PointX = pointX;
            this.PointY = pointY;
        }

        public double PointX { get; set; }

        public double PointY { get; set; }
    }
}

File: PHP/SoftUni/IntroToPhp/point.php
<?php

class Point {
    
    function __construct($pointX, $pointY) {
        $this->pointX = $pointX;
        $this->pointY = $pointY;
   }
    
    private $pointX;
    private $pointY;
    
    function getX() {
        return $this->pointX;
    }
    
    function getY() {
        return $this->pointY;
    }
}
